llvm/test/Instrumentation/AddressSanitizer/AMDGPU/asan_codegen_loop.ll

```diff
@@ -0,0 +1,504 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 3
+; RUN: opt  -passes=asan -S < %s | FileCheck %s --check-prefix=OPT
+; RUN: opt < %s -passes='asan,default<O3>' -o - | llc -O3 -mtriple=amdgcn-hsa-amdhsa -mcpu=gfx90a -o - | FileCheck %s --check-prefix=LLC-W64
```

Comment:
Is llc test is important?

I assume you can do IR only test, so it's in Instrumentation

llc should be on CodeGen side, but I am not convinced that they are needed.

Reply:
I am quite confident that llc tests are unnecessary.
You test should reflect what are you going to change in code.

And the change is about
```
const char kAMDGPUBallotName[] = "llvm.amdgcn.ballot.i64";
const char kAMDGPUUnreachableName[] = "llvm.amdgcn.unreachable";
```

which are visible in IR after https://github.com/llvm/llvm-project/pull/72247